### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by salihelfatih**
## Link Jira ticket:
https://jess-nguy.atlassian.net/browse/POZ-9

## Author Notes:

In your own words, what does this code change accomplish?
it's a frontend boilerplate for the team to build on and go back to JIC. 

## Impacts and Notes:

_Include notes about what the reviewer should consider when assessing the code changes._

## Testing Guidance:

_Write out how the reviewer could test the changes introduced in this PR._

## Author Checklist:

- [x] Tested locally.
- [x] Change Jira ticket status to Review.
- [ ] Updated the documentation.

### CODE REVIEW

**Review by jpdh9881 — CHANGES_REQUESTED** [review]
Please see updated instructions in the JIRA ticket

**Comment by jpdh9881** [comment]
everything looks great except that the .gitignore file seems to have been deleted in the main directory, allowing the node_modules folder in /server to be included in the git project. (the git ignore file tells git to ignore certain files and folders which should not be tracked in the repository. Including the node_modules folder makes the project unnecessarily big and can cause problems later on)
i think the best bet to fix this would be to:
1) switch to the main branch (git checkout main)
2) create a new POZ-9 branch (git checkout -b POZ-9-version-2)
3) install the vue project as before
4) finish the PR!

**Comment by Jess-Nguy** [comment]
You can also follow these steps to merge main into this branch
In a terminal with your path in this repo do the following
```
git checkout POZ-9      # gets you "on branch POZ-9"
git fetch origin        # gets you up to date with origin
git merge origin/main
```
Then fix all the merge conflict if there is any. It should show you what it is on main and what your current branch has. Remove the section of code you don't need and remove the git bracket details.
Please make sure you add reviewers to let people know you're ready for people to look at this Pull request.

File: README.md [hunk 1]
```diff
@@ -1,29 +1,32 @@
 # Pozter
-
 Applied Research Winter 2022 Project Pozter
-
-# Project setup:
```
Comment (Jess-Nguy) [inline README.md:5]: Please leave the previous readme details and add more if you'd like. The notes in the removed readme is important still.
It looks like you added your pull request details into the readme which shouldn't be here.

File: front-end/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# front-end
```
Comment (Jess-Nguy) [inline front-end/README.md:1]: Lets combine this readme with the root readme

**Review by Jess-Nguy — COMMENTED** [review]
I agree with with Joe said, let's revert the `.gitignore` so we don't have the node module  in the repo
to revert by commit do the following:
`git revert <commit number>`

**Pull request closed without merging**